### PR TITLE
Add card loaders and detailed card view

### PR DESCRIPTION
**Pull request by DJRHails**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a feature/bugfix branch** (right side). Don't request from master!
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

### Description
Screenshot 2019-11-21 at 22 21 51
Screenshot 2019-11-21 at 22 21 57
Screenshot 2019-11-21 at 23 27 19

### CODE REVIEW

**Comment by JamesDalboth** [comment]
I like the idea of displaying how many artifacts you own but I'm not sure the solution presented looks appealing, maybe too large of a box to just display how many artifacts you own? Also artifacts might not be the best phrase. Artpieces, Artworks maybe

File: app/src/providers/Web3Provider.tsx [hunk 1]
```diff
@@ -17,10 +17,11 @@ interface Web3ProviderProps {
 const Web3Context = React.createContext<Web3Interface>({} as any);
 
 export const Web3Provider: React.FC<Web3ProviderProps> = ({ networkId, accounts, children }) => {
-  const web3 = new Web3(Web3.givenProvider || 'ws://127.0.0.1:8545');
+  const { web3 } = window as any;
+  const _web3 = new Web3(web3.givenProvider || 'ws://127.0.0.1:8545');
```
Comment (JamesDalboth) [inline app/src/providers/Web3Provider.tsx]: We confident this works on rinkeby / heroku combo? Have had some troubles in the past

File: app/src/components/ArtworkItem.tsx [hunk 2]
```diff
@@ -1,67 +1,64 @@
...
-  render (): React.ReactNode {
-    if (!this.state) {
-      return 'Loading...';
-    }
+  if (!artwork) {
+    return <ArtworkCard img='https://file.globalupload.io/HO8sN3I2nJ.png'/>;
+  }
 
-    console.log('Artwork ' + JSON.stringify(this.state.artwork));
+  console.log('Artwork', artwork);
```
Comment (JamesDalboth) [inline app/src/components/ArtworkItem.tsx:35]: Was therebefore but should probably remove logging statement

**Review by JamesDalboth — CHANGES_REQUESTED** [review]
Few small things, main point is big artwork count box

File: app/src/components/ArtworkItem.tsx (hunk 2 above)
Comment (DJRHails) [reply app/src/components/ArtworkItem.tsx:35]: I'm actually pro this logging statement as it really highlights when you mucked up the reloading

File: app/src/providers/Web3Provider.tsx (hunk 1 above)
Comment (DJRHails) [reply app/src/providers/Web3Provider.tsx]: Only put this temporarily to make it work. You'll notice on resolved version it's much better

**Comment by DJRHails** [comment]
> I like the idea of displaying how many artifacts you own but I'm not sure the solution presented looks appealing, maybe too large of a box to just display how many artifacts you own? Also artifacts might not be the best phrase. Artpieces, Artworks maybe

Probably true but if we make that change we should make it everywhere not just here. We reference artifact too much in a lot of the ui.

As for the box I imagine we'd just intersperse it with the cards. If we had a lot more artifacts I imagine it'd look a lot nicer

**Review by JamesDalboth — APPROVED** [review]

**Pull request merged**